doc/whats_new/v1.0.rst

```diff
@@ -209,6 +209,13 @@ Changelog
   an adequate transformer.
   :pr:`18898` by :user:`Oras Phongpanagnam <panangam>`.
 
+:mod:`sklearn.cross_decomposition`
+..................................
+
+- |Fix| :class:`impute.IterativeImputer` now works
+  with :class:`cross_decomposition.PLSRegression` as estimator.
```

Comment:
I think it should rather be something like

> PLSRegression now properly returns a 1-dimensional prediction array for single-target prediction, instead of a 2d array. This allows compatibility with meta-estimators such as IterativeImpture.